swagger-spec/collection_submissions/list.yaml

```
@@ -0,0 +1,72 @@
+# /collection_submissions/{collection_submissions_id}/actions/
+get:
+  summary: Get a list of Collection Submission Actions for a Collection Submission
+  description: >-
+
+    This returns a list of the actions that have changed the Collection Submission moderation state.
+
+    #### Permissions
+
+    This information is only available to an admin on the Collection in question.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of nodes ids.
+
+    The `links` key contains a dictionary of links that can be used for [pagination](#tag/Pagination).
+  parameters:
+    - in: path
+      type: string
+      required: true
+      name: collection_submission_id
+      description: 'A short id for that collection submission'
+  tags:
+    - Collection Submissions
+  operationId: collection_submission_list
+  x-response-schema: Collection Submissions
+  consumes:
+    - application/json
+  responses:
+    '200':
+      description: 'OK'
+      examples:
+        application/json:
+          data:
+            id: 63bc18c66f4506000abae96f
```

Is this a list of submission? I'm not familiar with the formatting, but should it be the following format?
```suggestion
            - id: 63bc18c66f4506000abae96f
```